src/buildCommon.js

```diff
@@ -733,6 +733,8 @@ const staticSvg = function(value: string, options: Options): domTree.span {
     const svgNode = new domTree.svgNode([path], {
         "width": width + "em",
         "height": height + "em",
+        // Override CSS rule `.katex svg { width: 100% }`
+        "style": "width:" + width + "em",
         "viewBox": "0 0 " + 1000 * width + " " + 1000 * height,
         "preserveAspectRatio": "xMinYMin",
     });
```

Comment:
We should be able to get rid of this once we have non-combining `\vec` as well.  Then all of the non-stretchy accent code should use the same path.

Reply:
Agreed. That makes me feel better about the hacky duplication of `width`.